build.gradle

```diff
@@ -5,9 +5,9 @@ buildscript {
     }
 
     dependencies {
+        classpath 'com.google.errorprone:error_prone_core:2.15.0'
```

Comment:
Thoughts on removing this in order to pick up error_prone 2.15.0 from gradle-baseline?